### DISCUSSION TITLE
Help for stable helm chart version

### DISCUSSION THREAD

**Discussion by kalyanibharadwaj** [eclipse-ditto/ditto#2027] [Q&A]
When am trying to install ditto with helm chart in docker desktop, most of the pods going crash loop back with no proper logs.

tried helm version are  3.5.12 & 3.5.2.

suggest the working or stable helm version to spin up the Ditto cluster

**Comment by thjaeckle** [comment]
For locally starting Ditto, I recommend using Docker Compose instead of starting it in k8s via Helm.

If you have a proper k8s cluster, use Helm. I have no access to Docker Desktop and not much experience with it. Please share logs, otherwise there is no chance for us to help you.
